pkg/cluster/provider.go

```diff
@@ -105,6 +106,39 @@ func (p *Provider) Delete(name, explicitKubeconfigPath string) error {
 	return internaldelete.Cluster(p.logger, p.ic(name), explicitKubeconfigPath)
 }
 
+// Deletes one or more clusters created from list of cluster names and using Delete function.
+func (p *Provider) DeleteSome(clusters []string, explicitKubeconfigPath string) error {
```

Comment:
I don't think we should expand the API surface for this, all the CLI needs to do is a simple for loop calling p.Delete ?

Reply:
Thanks for pointing this out. I've moved the logic back into the CLI.